SIL.Scripture/VerseRef.cs

```diff
@@ -1338,6 +1337,27 @@ static void ParseVerseNumber(string vNum, out int number, out string segment)
 			segment = vNum.Substring(j);
 		}
 
+		/// <summary>
+		/// Parses a verse string and gets the leading numeric portion as a number.
+		/// Functionally identical to Verse.Set for Roman numbers, made distinct to preserve USX standard.
+		/// </summary>
+		/// <returns><c>true</c> if the entire string could be parsed as a single,
+		/// simple verse number in any supported script; <c>false</c> if the verse string represented
+		/// a verse bridge, contained segment letters, or was invalid</returns>
+		public bool TrySetVerseUnicode(string value)
+		{
+			{
+				verse = !TryGetVerseNum(value, out short vNum, false) ? value.Replace(rtlMark, "") : null;
```

Comment:
nit: why don't you directly use `out short verseNum`? Then the assignment in the next line wouldn't be necessary... (sorry, I overlooked that the first time...)

Reply:
Updated to use `out verseNum`. Using `short` would have declared a new variable, which would have conflicted with the member variable.